ibtk/include/ibtk/CCPoissonLevelRelaxationFACOperator.h

```diff
@@ -254,27 +254,27 @@ class CCPoissonLevelRelaxationFACOperator : public PoissonFACPreconditionerStrat
      *
      * \return A reference to this object.
      */
-    CCPoissonLevelRelaxationFACOperator& operator=(const CCPoissonLevelRelaxationFACOperator& that);
+    CCPoissonLevelRelaxationFACOperator& operator=(const CCPoissonLevelRelaxationFACOperator& that) = delete;
 
     /*
      * Level solvers and solver parameters.
      */
-    std::string d_level_solver_type, d_level_solver_default_options_prefix;
-    double d_level_solver_abs_residual_tol, d_level_solver_rel_residual_tol;
-    int d_level_solver_max_iterations;
-    std::vector<SAMRAI::tbox::Pointer<PoissonSolver> > d_level_solvers;
-    SAMRAI::tbox::Pointer<SAMRAI::tbox::Database> d_level_solver_db;
+    std::string d_level_solver_type = "", d_level_solver_default_options_prefix = "";
+    double d_level_solver_abs_residual_tol = 1.0e-50, d_level_solver_rel_residual_tol = 1.0e-5;
+    int d_level_solver_max_iterations = 1;
+    std::vector<SAMRAI::tbox::Pointer<PoissonSolver> > d_level_solvers = {};
```

Comment:
Another C++11-ism that we could go ahead and adopt while we are making 1000's of changes would be to convert "> >" to ">>". @drwells, what do you think?

Reply:
Yes, we should do that.

Reply:
@boyceg clang-format is correcting the `>>` back to `> >`. Should I go ahead and adjust this in the .clang-format file?